tests/sparseml/pytorch/sparsification/quantization/test_quantize.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from sparseml.pytorch.sparsification.quantization import (
+    QuantizationArgs,
+    QuantizationScheme,
+)
+
+
+@pytest.mark.parametrize(
+    "quantization_args_dict,target_quantization_args",
+    [
+        (
+            dict(num_bits=8, symmetric=False),
+            QuantizationArgs(num_bits=8, symmetric=False),
+        ),
+        (
+            dict(num_bits=4, symmetric=True),
+            QuantizationArgs(num_bits=4, symmetric=True),
+        ),
+        (
+            dict(num_bits=8, kwargs=dict(reduce_rage=True)),
+            QuantizationArgs(num_bits=8, kwargs=dict(reduce_rage=True)),
+        ),
+    ],
+)
+def test_quantization_args_from_dict(quantization_args_dict, target_quantization_args):
+    loaded_quantization_args = QuantizationArgs.parse_obj(quantization_args_dict)
```

Comment:
Could we rather create the object using kwargs, like `QuantizationArgs(**quantization_args_dict)` I don't think users will use pydantic's `parse_obj` to create `QuantizationArgs` objects

Reply:
Feel free to ignore this though

Reply:
end users will never create `QuantizationArgs` on their own - these classes are used to parse modifier arguments.

`parse_obj` is used because it will recursively parse any nested pydantic classes